Makefile

```diff
@@ -38,85 +38,141 @@ SERVICES_LIST := \
+
+# swarm
 export SWARM_STACK_NAME ?= simcore
+
+# version tags
 export DOCKER_IMAGE_TAG ?= latest
 export DOCKER_REGISTRY  ?= itisfoundation
 
 $(foreach v, \
-	SWARM_STACK_NAME DOCKER_IMAGE_TAG DOCKER_REGISTRY DOCKER_REGISTRY_NEW, \
+	SWARM_STACK_NAME DOCKER_IMAGE_TAG DOCKER_REGISTRY, \
 	$(info + $(v) set to '$($(v))'))
 
 
+.PHONY: help
+help: ## displays targets
+ifeq ($(IS_WIN),)
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+else
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "%-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+endif
+
+.DEFAULT_GOAL := help
+
+
+
 ## DOCKER BUILD -------------------------------
+#
+# - all builds are inmediatly tagged as 'local/{service}:${BUILD_TARGET}' where BUILD_TARGET='development', 'production', 'cache'
+# - only production and cache images are released (i.e. tagged pushed into registry)
+#
 TEMP_COMPOSE_YML := $(if $(IS_WIN)\
 	,$(shell (New-TemporaryFile).FullName)\
 	,$(shell mktemp -d /tmp/$(SWARM_STACK_NAME)-XXXXX)/docker-compose.yml)
 
 SWARM_HOSTS = $(shell $(DOCKER) node ls --format="{{.Hostname}}" 2>$(if $(IS_WIN),null,/dev/null))
 
+.PHONY: build
+build: .env ## Builds production images and tags them as 'local/{service-name}:production'
+	# Compiling front-end
+	@$(MAKE) -C services/web/client compile
+	# Building services
+	export BUILD_TARGET=production; \
+	$(DOCKER_COMPOSE) -f services/docker-compose.build.yml build --parallel
 
-create-stack-file: ## Creates stack file for production as $(output_file) e.g. 'make create-stack-file output_file=stack.yaml'
-	$(DOCKER_COMPOSE) -f services/docker-compose.yml -f services/docker-compose.prod.yml config > $(output_file)
 
-.PHONY: build
-build: .env .build-webclient ## Builds all core service images (user `make rebuild` to build w/o cache)
-	$(DOCKER_COMPOSE) -f services/docker-compose.yml build --parallel $(SERVICES_LIST)
+.PHONY: rebuild build-nc
+rebuild: build-nc
+build-nc: .env ## As build but w/o cache (alias: rebuild)
+	# Compiling front-end
+	@$(MAKE) -C services/web/client clean compile
+	# Building services
+	export BUILD_TARGET=production; \
+	$(DOCKER_COMPOSE) -f services/docker-compose.build.yml build --parallel --no-cache
 
-.PHONY: rebuild
-rebuild: .env .build-webclient
-	$(DOCKER_COMPOSE) -f services/docker-compose.yml build --no-cache --parallel $(SERVICES_LIST)
 
 .PHONY: build-devel
-build-devel: .env .build-webclient ## Builds images of core services for development
-	$(DOCKER_COMPOSE) -f services/docker-compose.yml -f services/docker-compose.devel.yml build --parallel
+build-devel: .env ## Builds development images and tags them as 'local/{service-name}:development'
+	# Compiling front-end ('compile' target needed since productions stage in Dockerfile of webserver copies client/tools/qooxdoo-kit/build:latest)
+	$(MAKE) -C services/web/client compile compile-dev
+	# Building services
+	export BUILD_TARGET=development; \
```

Comment:
Cmder doesn't like the "export" bit

Reply:
@odeimaiz follow [README.txt](README.txt) section on windows and it should work.